packages/app/src/sandbox/eval/transpiled-module.ts

```diff
@@ -971,7 +972,10 @@ export default class TranspiledModule {
       globals.__dirname = pathUtils.dirname(this.module.path);
       globals.__filename = this.module.path;
 
-      const exports = evaluate(
+      const exports = (this.module.path.indexOf('/node_modules/react/') === 0 ||
```

Comment:
@CompuIves A regex can be used instead, no? 🤔

```ts
const exports = this.module.path.match(/\/node_modules\/react(-dom)?\//)[0]
  ? evaluateScript
  : evaluate)(
  // ...
);
```

Reply:
Yeah this is super hacky, I don't even want to do that check there. It was purely for testing because it doesn't exist in dev.

In the final version this will probably be a different check, I'm not sure if I would opt for regex since this is a reaaaally hot code path, I'm not sure if I would do a check here 😂.

Reply:
No problem, just found it a bit verbose like this so wanted to help you out 🙂